### PR TITLE
Fix status labels for transactions vs orders.

### LINKED ISSUE #1353: Side bar: incorrect transaction status when Approve/Wrap/Unwrap token is failed
**Steps:** 
1. Run approve transaction 
2. Specify transaction conditions in the connected wallet to get an unsuccessful result (low gas limit or so)

3. wait till transaction fails

**AR**: approve token transaction status = expired
failed

the same is for the Wrap/Unwrap transactions
image

**ER**: approve token transaction status = failed.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- To go with release 1.1
- Fixes #1353 

### OLD
Screen Shot 2021-09-01 at 17 50 35

### NEW
Screen Shot 2021-09-01 at 18 00 55

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
Now all statuses look good to me 
image

I have to notice that I can't check 'Cancelled' order status due to #1373 issue.

**Comment by elena-zh** [comment]
'Cancelled' order status works fine.

**Review by alfetopito — APPROVED** [review]

File: src/custom/components/AccountDetails/Transaction/index.tsx [hunk 1]
```diff
@@ -382,21 +382,28 @@ export default function Transaction({ hash: id }: { hash: string }) {
...
               <SVG src={OrderOpenImage} description="Order Open" />
             )}
+
             {isPending
               ? 'Open'
               : isConfirmed && isTransaction
               ? 'Approved'
               : isConfirmed
               ? 'Filled'
+              : isExpired && isTransaction
```
Comment (anxolin) [inline src/custom/components/AccountDetails/Transaction/index.tsx:405]: nitpicking: there's a bit of logic repetition. In principle, you can do a pure javascript function, which receives the `ActivityType` and the `ActivityStatus` and they map it to a new type with all statuses. This new type could even be used to get the values from a static`STATUS_DETAILS` constant, so each state has it's label/image/description

```jsx
const statusUi = getStatusUi(status, type) // not sure how to call this, probably some of u come up with a better name
const { image, label, description } = STATUS_DETAILS[statusUi]

<SVG src={image} description={description} /> {label}
```

**Review by anxolin — APPROVED** [review]
Approved, however. Shouldn't this be somethig for 1.1? 

I would at least cherry-pick it to the branch

**Pull request merged**